third_party/istio-stable/install-config-istio.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
I can drop it since it's net new - right now it's just a mirror of `install-istio.sh` - but install-istio.sh should remain to keep some backwards support.